src/backend/base/langflow/components/memories/AstraDBMessageReader.py

```diff
@@ -63,20 +62,12 @@ def get_messages(self, **kwargs) -> list[Record]:
 
     def build(
         self,
-        session_id: Text,
+        session_id: str,
         collection_name: str,
         token: str,
         api_endpoint: str,
         namespace: Optional[str] = None,
     ) -> list[Record]:
-        try:
-            pass
-        except ImportError:
-            raise ImportError(
-                "Could not import langchain Astra DB integration package. "
-                "Please install it with `pip install langchain-astradb`."
-            )
```

Comment:
Import above - this is an artifact of a recent large refactoring

Reply:
Is this not needed anymore?

Reply:
Ah, good question. I think you're correct, I should move the import here actually. The `src/backend` poetry file does not directly import `langchain_astradb`, so for users running in backend-only mode, this check can be helpful. I'll re-add this